### ISSUE TITLE
Theme Duplication

### ISSUE THREAD

**Issue by denysvitali** [hugomodo/hugomodo-best-motherfucking-website#2]
Isn't this Hugo Theme similar to mine?  
You can see it live here: https://denv.it/blog/
https://github.com/denysvitali/hugo-thebestmotherfuckingblog

**Comment by thombruce** [comment]
Hey @denysvitali. Similar but different. I actually based it on your website before discovering that you'd also made that Hugo theme.

Have added links to the website as attribution, but happy to expand on the attribution if you want me to point people somewhere else. I would totally recommend your theme over mine if someone was looking for just a traditional drop-in-and-go Hugo theme, for example...

Sorry for the verbose response below. The TL;DR is this theme doesn't add much to my HugoModo "base theme", but I wanted something clean, simple and beautiful as the first theme I worked on while making sure the markup was on point... and The Best Motherfucking Website is fucking awesome!

Any more clarification needed, or if you want more/better attribution I'd be happy to oblige. Will leave this open for now to discuss.

**Why it's different**

As an example, here's the themes listing in config.toml for my HugoModo docs site (https://github.com/hugomodo/hugomodo-docs):

```toml
theme = [
  "hugomodo-cookie-consent",
  "hugomodo-fusejs",
  "hugomodo-shortcode-partials",
  "hugomodo-image-processing",
  "hugomodo-best-motherfucking-website",
  "hugomodo"
]
```

The base theme there is "HugoModo". On it's own it's mainly plain HTML (a bit like that _other_ motherfucking website), but it includes the bulk of the markup. In fact the HugoModo Best Motherfucking Website theme depends on it, and actually only has two partials itself: https://github.com/hugomodo/hugomodo-best-motherfucking-website/tree/master/layouts/partials

Everything above that is an "extension".

Basically it's my attempt to emulate a package manager for Hugo using Hugo's data files.

**Comment by denysvitali** [comment]
No problem! I wasn't looking for any kind of attribution, I was just genuinely interested in knowing how this theme differed from mine, and I'm happy to see that you're putting so much effort in making your own package-manager / plugin system 👍

I'll add your project to the hugo-theme README :rocket: 

Thanks for the clarification!
